grow/cache/podcache.py

```diff
@@ -105,9 +107,10 @@ def _write_json(self, path, obj):
         output = json.dumps(
             obj, cls=json_encoder.GrowJSONEncoder,
             sort_keys=True, indent=2, separators=(',', ': '))
-        temp_path = '{}.tmp'.format(path)
-        self._pod.write_file(temp_path, output)
-        self._pod.move_file_to(temp_path, path)
+        fp, temp_path = tempfile.mkstemp()
+        fp.write(output)
+        fp.close()
+        os.rename(temp_path, path)
```

Comment:
can you verify that `path` here is the full filesystem path and not the pod path?

Reply:
Updated to use `pod.abs_path`.